sycl/ReleaseNotes.md

```diff
@@ -16,6 +16,699 @@
+  instead. intel/llvm#16728
+  - Deprecated overloads were completely removed from the extension
+    specification. intel/llvm#14785
+- Deprecated current implementation of `get_backend_info` API. The SYCL 2020
+  specification currently does not document anything that could be queried
+  through it and therefore existing queries supported through it are deprecated
+  to avoid possible confusion. intel/llvm#16700
+
+### Upcoming API/ABI breakages
+
+This changes are available for preview under `-fpreview-breaking-changes` flag.
+They will be enabled by default (with no option to switch to the old behavior)
+in the next ABI-breaking release:
+
+- Removed implementation of `get_backend_info` APIs, see above in the
+  Deprecations section. intel/llvm#16700
+
+## Known Issues
+
+- SYCL headers use unreserved identifiers which sometimes cause clashes with
+  user-provided macro definitions (intel/llvm#3677). Known identifiers include:
+  - `G`. intel/llvm#11335
+  - `VL`. intel/llvm#2981
+- On Windows, the Unified Runtime's Level Zero leak check does not work
+  correctly with the default contexts on Windows. This is because on Windows
+  the release of the plugin DLLs races against the release of static global
+  variables (like the default context).
+- Intel Graphic Compiler's Vector Compute backend does not support
+  O0 code and often gets miscompiled, produces wrong answers
+  and crashes. This issue directly affects ESIMD code at O0. As a
+  temporary workaround, we have optimize ESIMD code even in O0 mode.
+  [00749b1e8](https://github.com/intel/llvm/commit/00749b1e8e3085acfdc63108f073a255842533e2)
+- When using `sycl_ext_oneapi_matrix` extension it is important for some
+  devices to use the sm version (Compute Capability) corresponding to the
+  device that will run the program, i.e. use `-fsycl-targets=nvidia_gpu_sm_xx`
+  during compilation. This particularly affects matrix operations using
+  `half` data type. For more information on this issue consult with
+  https://docs.nvidia.com/cuda/cuda-c-programming-guide/index.html#wmma-restrictions
+- C/C++ math built-ins (like `exp` or `tanh`) can return incorrect results
+  on Windows for some edge-case input. The problems have been fixed in the
+  SYCL implementation, and the remaining issues are thought to be in MSVC.
+- There are known issues and limitations in virtual functions
+    functionality, such as:
+    - Optional kernel features handling implementation is not complete yet.
+    - AOT support is not complete yet.
+    - A virtual function definition and definitions of all kernels using it
+    must be in the same translation unit. Please refer to
+    [`sycl/test-e2e/VirtualFunctions`](https://github.com/intel/llvm/tree/b23d69e2c3fda1d69351137991897c96bf6a586d/sycl/test-e2e/VirtualFunctions)
+    to see the list of working and non-working examples.
+
+
+43ee65117935 [UR] Fix potential deadlock in the WaitEvent path of CmdBuffers (#16697)
```

Comment:
@intel/unified-runtime-reviewers, I haven't looked into those items at all. My plan is to drop them completely, but if you know of something that is worth mentioning, then please let me know and I will preserve those items.

Reply:
As for all the `[UR]` tagged changes, none of them are not user facing so I think they should be removed. UR should be mostly invisible, the interesting things a fixes in backends, e.g. L0.